pyexchange/exchange2010/soap_request.py

```diff
@@ -7,6 +7,10 @@
 from lxml.builder import ElementMaker
 from ..utils import convert_datetime_to_utc
 
+import sys
+if sys.version_info[0] == 3:
+	unicode = str
```

Comment:
Hey, thanks for the commit. Can you tell me a little more why this was necessary? What broke?